app/controllers/posts_controller.rb

```diff
@@ -8,4 +8,29 @@ def show
     @user = User.find(params[:user_id])
     @post = @user.posts.find(params[:id])
   end
+
+  def new
+    @post = Post.new
+  end
+
+  def create
+    puts 'Current User'
```

Comment:
- [ ] Kindly remove this print statement. It is generally not a good practice to print statements. Kindly cleanup your code.